### PR TITLE
DOC-1225 RS reorg: Databases section

### PR DESCRIPTION
**Pull request by kaitlynmichael**
First section of RS reorg effort: Databases. 
Everything that is created, edited, recovered, imported, exported, or backed up on a database level (besides Security and Networking) has been moved into this section. 
[Staged preview](https://docs.redislabs.com/staging/jira-doc-1225/rs/databases/)
Jira epic: DOC-714
Jira ticket: DOC-1225

### CODE REVIEW

File: content/rs/databases/create-database.md [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
+aliases: [
+    /rs/administering/creating-databases/_index.md,
+    /rs/administering/database-operations/creating-database/,
+    /rs/databases/create-database.md,
+    /rs/databases/create-database/,
+  
+]
 ---
-You can create Redis databases that are are clustered and distributed across a single Redis Enterprise Software (RS) cluster.
+Redis Enterprise Software give you the ability to create databases that are are clustered and distributed across a single cluster.
```
Comment (banker) [inline content/rs/databases/create-database.md:17]:
```suggestion
Redis Enterprise Software give you the ability to create databases that are are clustered and distributed across a single cluster.
```
```suggestion
Redis Enterprise Software lets you create databases and distribute them across a cluster of nodes.
```

Comment (banker) [inline content/rs/databases/create-database.md:17]:
```suggestion
Redis Enterprise Software give you the ability to create databases that are are clustered and distributed across a single cluster.
```
```suggestion
Redis Enterprise Software lets you create databases and distribute them across a cluster of nodes.
```

File: content/rs/databases/active-active/get-started-active-active.md [hunk 2]
```diff
@@ -1,29 +1,34 @@
...
-To run an Active-Active database on installations from the [RS download package]({{< relref "/rs/getting-started/_index.md" >}}),
+To run an Active-Active database on installations from the [RS download package]({{< relref "rs/installing-upgrading/get-started-redis-enterprise-software.md" >}}),
 set up two RS installations and continue from Step 2.
 
 {{< note >}}
 This getting started guide is for development or demonstration environments.
 To set up an Active-Active database in a production environment, use the instructions for
-[creating an Active-Active database]({{< relref "/rs/administering/creating-databases/create-active-active.md" >}}).
+[creating an Active-Active database]({{< relref "/rs/databases/active-active/create-active-active.md" >}}).
 {{< /note >}}
```
Comment (rrelledge) [inline content/rs/databases/active-active/get-started-active-active.md:33]: Maybe change the following headers:
- "Connecting using redis-cli" to "Connect with redis-cli"
- "Connecting using Hello World application in Python"  to "Connect with Python"

There's also some instances where Redis and Python are not capitalized properly.

File: content/rs/databases/active-active/create-active-active.md [hunk 3]
```diff
@@ -26,7 +32,7 @@ Every instance of an Active-Active database can receive write operations, and al
 
 - Two or more machines with the same version of RS installed
 - Network connectivity and cluster FQDN name resolution between all participating clusters
-- [Network time service]({{< relref "/rs/administering/designing-production/active-active#network-time-service-ntp-or-chrony" >}}) listener (ntpd) configured and running on each node in all clusters
+- [Network time service]({{< relref "/rs/databases/active-active#network-time-service-ntp-or-chrony" >}}) listener (ntpd) configured and running on each node in all clusters
 
 ## Creating an Active-Active database
```
Comment (rrelledge) [inline content/rs/databases/active-active/create-active-active.md:37]:
```suggestion
## Create an Active-Active database
```

File: content/rs/databases/active-active/create-active-active.md [hunk 4]
```diff
@@ -73,7 +79,7 @@ Every instance of an Active-Active database can receive write operations, and al
         When replication is enabled, every Active-Active database master shard is replicated to a corresponding replica shard. The replica shards are then used to synchronize data between the instances, and the master shards are dedicated to handling client requests.
         We also recommend that you enable [replica HA]({{< relref "/rs/administering/database-operations/replica-ha.md" >}}) to ensure that the replica shards are highly-available for this synchronization.
```
Comment (rrelledge) [inline content/rs/databases/active-active/create-active-active.md:81]: Similar to "Get started with A-A", maybe change the following headers:
- "Connecting using redis-cli" to "Connect with redis-cli"
- "Connecting using Hello World application in Python" to "Connect with Python"

There are also places where Redis and Python are not capitalized correctly.

File: content/rs/databases/active-active/edit-aa-database.md [hunk 5]
```diff
@@ -1,24 +1,14 @@
...
-    The database settings appear.
-1. Change the any of the [configurable database settings]({{< relref "/rs/administering/creating-databases/_index.md" >}}).
-
-    {{< note >}}
-For [Active-Active database instances](#updating-crdb-configuration), most database settings only apply to the instance that you are editing.
-    {{< /note >}}
-
-1. Click **Update**.
 
 ## Editing Active-Active database configuration
```
Comment (rrelledge) [inline content/rs/databases/active-active/edit-aa-database.md:13]:
```suggestion
## Edit Active-Active database configuration
```

File: content/rs/databases/active-active/delete-aa-database.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    /rs/databases/active-active/delete-aa-database.md,
+    /rs/databases/active-active/delete-aa-database/,
+]
+---
+
+When you delete an Active-Active database (formerly known as CRDB),
+all instances of the Active-Active database are deleted from all participating clusters.
+
+{{% warning %}}
+This action is an immediate, non-reversible, and has no rollback.
```
Comment (rrelledge) [inline content/rs/databases/active-active/delete-aa-database.md:19]:
```suggestion
This action is immediate, non-reversible, and has no rollback.
```

File: content/rs/databases/active-active/delete-aa-database.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+{{% /warning %}}
+
+Because Active-Active databases are made up of instances on multiple participating clusters,
+to restore a deleted Active-Active database you must create the database again with all of its instances
+and then restore the data to the database from backup.
+
+We recommended that you:
+
+- Backup your data and test the restore on another database before you delete an Active-Active database.
```
Comment (rrelledge) [inline content/rs/databases/active-active/delete-aa-database.md:28]:
```suggestion
- Back up your data and test the restore on another database before you delete an Active-Active database.
```

File: content/rs/databases/active-active/app-failover-active-active.md [hunk 8]
```diff
@@ -1,13 +1,18 @@
...
+weight: 60
 alwaysopen: false
 categories: ["RS"]
-aliases: /rs/developing/crdbs/region-application-failover-active-active/
+aliases: [
+    /rs/developing/crdbs/region-application-failover-active-active/,
+    /rs/databases/active-active/app-failover-active-active.md,
+    /rs/databases/active-active/app-failover-active-active/,
+]
+
```
Comment (rrelledge) [inline content/rs/databases/active-active/app-failover-active-active.md:13]: You can change the headers to sentence case and maybe change "Detecting Failure" to "Detect failure".

File: content/rs/databases/active-active/data-types/_index.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+]
+---
+
+
+Active-Active databases use conflict-free replicated data types (CRDTs). From a developer perspective, most supported data types work the same for Active-Active and standard Redis databases. However, a few methods also come with specific requirements in Active-Active databases.
+
+Even though they look identical to standard Redis data types, there are specific rules that govern the handling of
+conflicting concurrent writes for each data type.
+
+As conflict handling rules differ between data types, some commands have slightly different requirements in Active-Active databases vs standard Redis databases.
```
Comment (rrelledge) [inline content/rs/databases/active-active/data-types/_index.md:22]:
```suggestion
As conflict handling rules differ between data types, some commands have slightly different requirements in Active-Active databases versus standard Redis databases.
```

File: content/rs/databases/active-active/data-types/lists-active-active.md [hunk 10]
```diff
@@ -1,13 +1,19 @@
 ---
-Title: Developing with Lists in an Active-Active Database
+Title: Lists in Active-Active Databases
```
Comment (rrelledge) [inline content/rs/databases/active-active/data-types/lists-active-active.md:2]:
```suggestion
Title: Lists in Active-Active databases
```

File: content/rs/databases/active-active/data-types/lists-active-active.md [hunk 11]
```diff
@@ -1,13 +1,19 @@
...
+    /rs/references/developing-for-active-active/developing-lists-active-active.md,
+    /rs/databases/active-active/data-types/lists-active-active.md,
+    /rs/databases/active-active/data-types/lists-active-active/,
+]
 ---
-Redis Lists are simply lists of strings, sorted by insertion order. It
+Redis lists are simply lists of strings, sorted by insertion order. It
 is possible to add elements to a Redis List that push new elements to
 the head (on the left) or to the tail (on the right) of the list. Redis
 Lists can be used to easily implement queues (using LPUSH and RPOP, for
```
Comment (rrelledge) [inline content/rs/databases/active-active/data-types/lists-active-active.md:19]:
```suggestion
the head (on the left) or to the tail (on the right) of the list. Redis
lists can be used to easily implement queues (using LPUSH and RPOP, for
```

File: content/rs/databases/active-active/data-types/sets-active-active.md [hunk 12]
```diff
@@ -1,11 +1,17 @@
...
-aliases: /rs/developing/crdbs/developing-sets-crdb/
+aliases: [
+    /rs/developing/crdbs/developing-sets-crdb/,
+    /rs/references/developing-for-active-active/developing-sets-active-active/,
+    /rs/references/developing-for-active-active/developing-sets-active-active.md
+    /rs/databases/active-active/data-types/sets-active-active.md,
+    /rs/databases/active-active/data-types/sets-active-active/,
+]
 ---
 Redis Sets are an unordered collection of Strings. It is possible to
```
Comment (rrelledge) [inline content/rs/databases/active-active/data-types/sets-active-active.md:16]:
```suggestion
A Redis set is an unordered collection of strings. It is possible to
```

File: content/rs/databases/active-active/data-types/strings-active-active.md [hunk 13]
```diff
@@ -1,12 +1,18 @@
 ---
-Title: Strings and Bitfields with Active-Active Databases
+Title: Strings and Bitfields in Active-Active databases
```
Comment (rrelledge) [inline content/rs/databases/active-active/data-types/strings-active-active.md:2]:
```suggestion
Title: Strings and bitfields in Active-Active databases
```

**Review by rrelledge — APPROVED** [review]
This reorg looks good! I left suggestions for some minor things that I noticed.

**Review by lanceleonard — APPROVED** [review]
LGTM

**Pull request merged**